doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -88,6 +92,13 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
 SSL_CIPHER_is_aead() returns 1 if the cipher B<c> is AEAD (e.g. GCM or
 ChaCha20/Poly1305), and 0 if it is not AEAD.
 
+SSL_CIPHER_find() returns a B<SSL_CIPHER> structure which has the cipher ID B<ptr>.
+The B<ptr> parameter is a two elements array of B<char>, which stores the two bytes
+TLS cipher ID in network byte order. This parameter is usually retrieved from a TLS
+packet by using functions like L<PACKET_get_bytes(3)>.
+
+SSL_CIPHER_get_id() returns the ID of the given cipher B<c>.
```

Comment:
The returned ID here is an OpenSSL-specific value (related to, but not equal to, the IANA-specified ciphersuite value, for TLS ciphers).  This is something of a historical artifact dating to SSLv2 support.